crates/db_views_actor/src/structs.rs

```diff
@@ -42,6 +43,7 @@ pub struct CommunityView {
   pub subscribed: SubscribedType,
   pub blocked: bool,
   pub counts: CommunityAggregates,
+  pub languages: Vec<Language>
```

Comment:
Same, move this to the response.

Reply:
This is pretty easy for `CommunityResponse`, but theres also this:
```
pub struct ListCommunitiesResponse {
  pub communities: Vec<CommunityView>,
}
```

I could change it like one of the following, but either option seems rather confusing.
```
pub struct ListCommunitiesResponse {
  pub communities: Vec<CommunityView>,
  pub discussion_languages: Vec<Vec<LanguageId>>
}
```

or 
```
pub struct ListCommunitiesResponse {
  pub communities: Vec<CommunityView, Vec<LanguageId>>,
}
```

Reply:
Why does the list of languages for each community need to come back when listing a lot of communities? Shouldn't only filtering be necessary?

Reply:
Ah, makes sense that languages are only needed in GetCommunityResponse. Added that.